src/modules/did/did.service.ts

```diff
@@ -85,6 +83,10 @@ export class DIDService {
     }
   }
 
+  async onModuleInit() {
+    await this.InitEventListeners();
+  }
```

Comment:
Is this a nestjs best practice? Or was this done to fix an issue you encountered perhaps?